app/views/shared/_spinner_button.html.erb

```diff
@@ -12,10 +12,10 @@ locals:
 <%= tag.div class: classes do %>
   <div class="spinner-button__content">
     <%= content %>
-    <span class="spinner-button__spinner" aria-hidden="true">
-      <span class="spinner-button__spinner-dot"></span>
-      <span class="spinner-button__spinner-dot"></span>
-      <span class="spinner-button__spinner-dot"></span>
+    <span class="spinner-dots spinner-dots--centered text-white" aria-hidden="true">
```

Comment:
Ideally this should be its own component, like a ViewComponent counterpart to the React component introduced in the branch.

Since this is already growing quite large, might save that for a follow-up.